gcs/deleteFile/deleteFile.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+//
+// DeleteFile is a probe that deletes a file from GCS.
+// It tests the file deletion aspect of GCS and ensures that the file
+// is deleted after the deletion request is made.
+
+package gcs.deletefile
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/google/cloudprober/logger"
+	"github.com/google/cloudprober/metrics"
+	"github.com/google/cloudprober/probes/options"
+	"github.com/google/cloudprober/targets/endpoint"
+
+	"cloud.google.com/go/storage"
+)
+
+// Probe holds aggregate information about all probe runs, per-target.
+type Probe struct {
+	name    string
+	c       *ProbeConf
```

Comment:
Calling this `config` or even `conf` would make it much clearer what's going on below.

Reply:
I have removed the deleteFile.go from this PR, as I want to merge it in a different PR.